### ISSUE TITLE
EEG-inception MI results

### ISSUE THREAD

**Issue by Norman-GM** [braindecode/braindecode#435]
As the decription in EEG-inception`[1]`, it has the accuracy 88.39% even without the augumentation, but it has **no description of Preprocessing steps** 
In my attempt on BCIC-IV 2a dataset with the default preprocessing tutorial of [2](https://braindecode.org/stable/auto_examples/plot_how_train_test_and_tune.html), the 5-fold model validation acc is 57.64%±8.70% ，seems has the overfit problem. How should I prepocrssing the origin data, and reproduce the results in the paper?
Looking forward your reply, thanks!
[1] Zhang, C., Kim, Y. K., & Eskandarian, A. (2021). EEG-inception: an accurate and robust end-to-end neural network for EEG-based motor imagery classification. Journal of Neural Engineering, 18(4), 046014.

**Comment by bruAristimunha** [comment]
Hi @NormanGod,

Thank you for using braindecode!

This could be happening for several reasons, and unfortunately, it is a huge part of the challenge of open-source science. That is why the authors must publish the codes in some reproducible framework, and reproductions usually report between 15~5% less than the authors write. 

I can help you, but I need more information. 

In reality, the EEG-Inception has two homonym papers, one for ERP and another for Motor Imagery, from different authors. Here, we have the implementation for both, but only in the more recent version and not released. 
 
What version of braindecode do you have installed? To install the version directly from the repository, you will need to install it with the command:

```bash
pip install -U https://api.github.com/repos/braindecode/braindecode/zipball/master
```
Try to import [EEGInceptionMI](
https://braindecode.org/dev/generated/braindecode.models.EEGInceptionMI.html#braindecode.models.EEGInceptionMI) to ensure that you are using the correct model. Could you also provide our code? It could be a minor bug that is hard to catch, even copying some braindecode tutorial. I recommend trying to remove the pre-processing step if your goal is reproduction.

Also, a disclaimer, in models that the braindecode team has reimplemented, there is no guarantee that they are entirely correct. We always try our best, but several components can impact the final result. Unfortunately, the core team have limited time for extensive studies to reproduce every choice not reported in the article.

**Comment by NormanGod** [comment]
Hi @bruAristimunha thanks for taking a look!

## Here are my code for reproduction
[https://colab.research.google.com/drive/1Oav9DE0vXdBh6UTLfBoegZYYTTyyfftV#scrollTo=GWU1fdumwaOG](https://colab.research.google.com/drive/1Oav9DE0vXdBh6UTLfBoegZYYTTyyfftV#scrollTo=GWU1fdumwaOG)
Since it has no GPU, I training on my own PC with same code. the result of 5-fold validation on first person is in the attachment zip.

[reproduce_EEG_inceptionMI.zip](https://github.com/braindecode/braindecode/files/10791580/reproduce_EEG_inceptionMI.zip)
## Validation setup

We want to use the validation scheme like [How to train test and tune] (https://braindecode.org/stable/auto_examples/plot_how_train_test_and_tune.html) of option 3. But utilize the validation model with the highest validation accuracy and test session_E only once. How should the best model be selected, and what changes should be made to the code?

## Shuffle problem

We utilized to divide the train dataset into numerous epochs (3s in BCIC IV -2a), shuffle epochs (between epochs, not within each epoch ), and then use kfold validation to distinguish training data from validation data. Is this correct? or may result in time series data leaky? If we used a longer epoch, would this problem be resolved?

## Reproduction problem

As the description in code, the author mention their hyper parameters and preprocessing settings in details, only say they use Adam optimizers with the learning rate of 5e-3 and batch size of 32.

In result of k-fold cross-session on subject 1, it seems can not reach the same performance mention in paper, which part might be wrong?

**looking forward to your reply, thanks.**

**Comment by bruAristimunha** [comment]
Hi @NormanGod,

Unfortunately I can't access the colab, can you change the view option to public?

I also applied for access three days ago

**Comment by NormanGod** [comment]
Hi @bruAristimunha,
Sorry for the inconvenience :( , please try this new webpage: https://colab.research.google.com/drive/1YKnwBKpTFyTlH0Id6nk_VhfREy-eSgv9?usp=share_link
